akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -25,6 +25,12 @@ to route the message with the entity id to the final destination.
 Cluster sharding will not be active on members with status :ref:`WeaklyUp <weakly_up_java>` 
 if that feature is enabled.
 
+.. warning::
+   **Don't use Cluster Sharding together with Automatic Downing**,
```

Comment:
Ok, nevermind - i see that docs section we link to here has a link now, that's good